controller/src/main/java/org/apache/rocketmq/controller/ControllerManager.java

```diff
@@ -93,6 +100,8 @@ protected <T> RunnableFuture<T> newTaskFor(final Runnable runnable, final T valu
                 return new FutureTaskExt<T>(runnable, value);
             }
         };
+        this.notifyService = new NotifyService();
+        this.notifyService.initialize();
```

Comment:
How about move  `this.notifyService.initialize();`  to ControllerManager initialize method?